packages/apollo-codegen-swift/src/language.ts

```diff
@@ -29,7 +29,23 @@ export interface Property {
 }
 
 export function escapedString(string: string) {
-  return string.replace(/"/g, '\\"').replace(/\n/g, "\\n");
+  if (string.includes('"""')) {
+    // This includes a multi-line string literal, and we may strip out meaningful
+    // whitespace if we try to strip whitespace. Don't try.
+    return string.replace(/"/g, '\\"').replace(/\n/g, "\\n");
+  } else {
+    // Strip unnecessary whitespace.
+    var lines = string
```

Comment:
`var` is so 2000-late, the hip thing nowadays is `const` or `let` (which is precisely opposite swift's `let`). ;)

Reply:
> which is precisely opposite swift's `let`). ;)

This is why I drink.